apps/sv/frontend/src/__tests__/governance/forms/set-dso-rules-form.test.tsx

```diff
@@ -385,3 +388,146 @@ describe('Set DSO Config Rules Form', () => {
     expect(screen.queryByTestId('config-diffs-display')).toBeDefined();
   });
 });
+
+describe('Next Scheduled Synchronizer Upgrade', () => {
```

Comment:
These tests are a bit difficult to follow without context so I've added some comments to make it easier